docker.go

```diff
@@ -100,6 +101,15 @@ func (p Plugin) Exec() error {
 		fmt.Println("Registry credentials not provided. Guest mode enabled.")
 	}
 
+	// pre-pull cache image
+	if p.Build.CacheFrom != "" {
```

Comment:
Why pre-pulling it ? you are going to slow the plugin no ?

Reply:
oh no nvm

Reply:
But I'm not sure if it's useful with --pull param

Reply:
`--cache-from` gets ignored if the image is not local

Reply:
It's actually an array, I'll push an update.